_data/companies/opslock.yml

```diff
@@ -1,7 +1,7 @@
 ---
 name: Opslock
-wfh: Strongly Encouraged
-travel: Discouraged
-visitors: Discouraged
-events: Discouraged
-last_update: 2020-03-15
+wfh: Required
+travel: Required
```

Comment:
Should this say forbidden?

Reply:
Wow, thanks for double-checking this. I've made the fix.